tests/cstor/cspc/positive/provisioning/cspc_provisioning_utils_test.go

```diff
@@ -85,6 +85,21 @@ func verifyDesiredCSPICount() {
 	Expect(cspiCount).To(Equal(3), "Mismatch Of CSPI Count")
 }
 
+func verifyDesiredCSPICountTo(count int) {
```

Comment:
Both the functions look similar in functionality by name. Can we have this renamed to `verifyDesiredHealthyCSPICount` ?

Reply:
The difference is -- one takes arg and other does not.

Reply:
I see both `verifyDesiredCSPICountTo` and `verifyDesiredCSPIResourceCountTo` take `count int` as arg.

Reply:
One takes count for `healthy` and one just for CSPIs irrespective of state. 
Anyways -- will refactor those funtions in subsequent PRs